<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regression</class>
 <widget class="QMainWindow" name="Regression">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="ButtonMenu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>80</width>
      <height>22</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>geht wieder zur√ºck zur Rechnerauswahl</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(54, 54, 54)</string>
    </property>
    <property name="text">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEingabe">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAusgabe">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>450</y>
      <width>651</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainPunkte">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAnzeige">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineBestimmheitsmass">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>181</width>
      <height>154</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioPolynom">
       <property name="text">
        <string>Polynomisch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioExponential">
       <property name="text">
        <string>Exponentiell</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioLogarithmus">
       <property name="text">
        <string>Logarithmisch</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioPotenz">
       <property name="text">
        <string>Potenz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioLn_beliebig">
       <property name="text">
        <string>Logarithmisch (beliebige Basis)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioExp_beliebig">
       <property name="text">
        <string>Exponentiell (beliebige Basis)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="lineBasis">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
